### ISSUE TITLE
Fedora build - no sub-menu items for the Add menu items

### ISSUE THREAD

**Issue by planmac** [mkeeter/antimony#152]
Only able to add a script - no other items available in the 2D, 3D etc. menu items

**Comment by mcarlo95** [comment]
Known bug. Same problem here

**Comment by imperfectlink** [comment]
Same problem here. F23 Gnome desktop.

**Comment by mkeeter** [comment]
Sounds pretty reproducible – I'll spin up a Fedora VM and see if I can solve this for the upcoming 0.9.1 release.

**Comment by mkeeter** [comment]
I'm having trouble reproducing this issue.

I built on Fedora 23, then ran the app from the build folder (i.e. `./app/antimony`), and all of the nodes showed up.  @imperfectlink, how are you invoking the application?

**Comment by sagitter** [comment]
Antimony is runned by `antimony %f` command; `Add` menu is totally invisible to me.

**Comment by mkeeter** [comment]
Did you build it from source or install it from the Fedora package?

**Comment by sagitter** [comment]
> Did you build it from source or install it from the Fedora package?

I'm the Fedora packager of Antimony; latest Antimony (0.9.1c) code has been compiled and packed as RPM.

**Comment by mkeeter** [comment]
Thanks for updating the package – I'll install from there and see if I can debug.

**Comment by sagitter** [comment]
It's not available online yet. Let me rebuild it for you.

**Comment by sagitter** [comment]
> Thanks for updating the package – I'll install from there and see if I can debug.

Please, use Fedora 23 i686 if you can; exists a bug of `boost-python3` on Fedora 23 x86_64.
http://copr-fe.cloud.fedoraproject.org/coprs/sagitter/Ipopt-EPEL/build/316346/

**Comment by imperfectlink** [comment]
I'm just calling it up with `antimony` in the terminal. No errors, just odd behaviour.

I installed using `dnf install antimony --enablerepo=updates-testing`

I removed and installed the i686 package from here: [](http://koji.fedoraproject.org/koji/buildinfo?buildID=699437)

Here is the output:
```
QGtkStyle could not resolve GTK. Make sure you have installed the proper libraries.
QGtkStyle could not resolve GTK. Make sure you have installed the proper libraries.
libGL error: unable to load driver: swrast_dri.so
libGL error: failed to load driver: swrast
Unrecognized OpenGL version
Unrecognized OpenGL version
```

**Comment by awgrover** [comment]
Same problem.

**Comment by mkeeter** [comment]
I don't have the resources to support prebuilt packages for all the flavors of Linux, but will support folks using the app (on Mac) and building from source (on Linux).

@awgrover, I'm in the process of switching from `qmake` to `cmake`, which will hopefully make the build / bundle / deploy process more robust – I'll ping you when it's ready for testing.

**Comment by mkeeter** [comment]
@awgrover / @sagitter 
I've updated everything to use `cmake`, and clarified the [building instructions](https://github.com/mkeeter/antimony/blob/develop/BUILDING.md#linux)

When you build the application, it will copy `sb/nodes` and `sb/fab` into the same folder as `antimony` (usually `Antimony/build/app/`).  This search path is used when `antimony` is called from the build folder.

On startup, the application looks for `sb/nodes`; if this folder doesn't exist, it assumes that it has been installed (to `/usr/local/bin/antimony` with `sudo ninja install`) and looks in `/usr/local/share/antimony/nodes` instead.

(this is all Linux-specific; on Mac, I just put `nodes` and `fab` into `Antimony.app/Resources`)

**Comment by sagitter** [comment]
Hi @mkeeter,

>I've updated everything to use cmake, and clarified the building instructions

Are your changes for next release ?

**Comment by mkeeter** [comment]
@sagitter 
I don't know how Fedora usually handles packaging (whether they prefer a tagged release or building from `develop` is fine).  If they prefer a specific release number, I'll make a few more updates and push out 0.9.2.

**Comment by sagitter** [comment]
> I don't know how Fedora usually handles packaging (whether they prefer a tagged release or building from develop is fine). If they prefer a specific release number, I'll make a few more updates and push out 0.9.2.

I think, it is a modification sufficiently important to justify a new release; of course, it is not fundamental for Fedora packaging.

**Comment by sagitter** [comment]
`Add` menu is still unvisible unfortunately.
`antimony 0.9.2` files installed (Fedora):

```
/usr/bin/antimony
/usr/lib/python3.5/site-packages/fab
/usr/lib/python3.5/site-packages/fab/__init__.py
/usr/lib/python3.5/site-packages/fab/__pycache__
/usr/lib/python3.5/site-packages/fab/__pycache__/__init__.cpython-35.opt-1.pyc
/usr/lib/python3.5/site-packages/fab/__pycache__/__init__.cpython-35.pyc
/usr/lib/python3.5/site-packages/fab/__pycache__/shapes.cpython-35.opt-1.pyc
/usr/lib/python3.5/site-packages/fab/__pycache__/shapes.cpython-35.pyc
/usr/lib/python3.5/site-packages/fab/__pycache__/types.cpython-35.opt-1.pyc
/usr/lib/python3.5/site-packages/fab/__pycache__/types.cpython-35.pyc
/usr/lib/python3.5/site-packages/fab/shapes.py
/usr/lib/python3.5/site-packages/fab/types.py
/usr/share/antimony
/usr/share/antimony/nodes
/usr/share/antimony/nodes/2D
/usr/share/antimony/nodes/2D → 3D
/usr/share/antimony/nodes/2D → 3D/Revolve
/usr/share/antimony/nodes/2D → 3D/Revolve/revolve_xy_x.node
/usr/share/antimony/nodes/2D → 3D/Revolve/revolve_xy_y.node
/usr/share/antimony/nodes/2D → 3D/extrude.node
/usr/share/antimony/nodes/2D → 3D/loft.node
/usr/share/antimony/nodes/2D/Circles
/usr/share/antimony/nodes/2D/Circles/circle_center.node
/usr/share/antimony/nodes/2D/Circles/circle_edge.node
/usr/share/antimony/nodes/2D/Rectangles
/usr/share/antimony/nodes/2D/Rectangles/rectangle_center.node
/usr/share/antimony/nodes/2D/Rectangles/rectangle_center_rounded.node
/usr/share/antimony/nodes/2D/Rectangles/rectangle_corner.node
/usr/share/antimony/nodes/2D/Rectangles/rectangle_corner_rounded.node
/usr/share/antimony/nodes/2D/Triangles
/usr/share/antimony/nodes/2D/Triangles/triangle.node
/usr/share/antimony/nodes/2D/Triangles/triangle_right.node
/usr/share/antimony/nodes/2D/point.node
/usr/share/antimony/nodes/2D/polygon.node
/usr/share/antimony/nodes/2D/text.node
/usr/share/antimony/nodes/3D
/usr/share/antimony/nodes/3D → 2D
/usr/share/antimony/nodes/3D → 2D/slice_xy.node
/usr/share/antimony/nodes/3D → 2D/slice_xz.node
/usr/share/antimony/nodes/3D → 2D/slice_yz.node
/usr/share/antimony/nodes/3D/Cones
/usr/share/antimony/nodes/3D/Cones/cone.node
/usr/share/antimony/nodes/3D/Cones/cone_edge.node
/usr/share/antimony/nodes/3D/Cones/cone_height.node
/usr/share/antimony/nodes/3D/Cubes
/usr/share/antimony/nodes/3D/Cubes/cube_center.node
/usr/share/antimony/nodes/3D/Cubes/cube_center_rounded.node
/usr/share/antimony/nodes/3D/Cubes/cube_corner.node
/usr/share/antimony/nodes/3D/Cubes/cube_corner_rounded.node
/usr/share/antimony/nodes/3D/Cylinders
/usr/share/antimony/nodes/3D/Cylinders/cylinder_x.node
/usr/share/antimony/nodes/3D/Cylinders/cylinder_y.node
/usr/share/antimony/nodes/3D/Cylinders/cylinder_z.node
/usr/share/antimony/nodes/3D/Pyramids
/usr/share/antimony/nodes/3D/Pyramids/pyramid_center.node
/usr/share/antimony/nodes/3D/Pyramids/pyramid_edge.node
/usr/share/antimony/nodes/3D/Spheres
/usr/share/antimony/nodes/3D/Spheres/sphere_center.node
/usr/share/antimony/nodes/3D/Spheres/sphere_edge.node
/usr/share/antimony/nodes/3D/Toruses
/usr/share/antimony/nodes/3D/Toruses/torus_x.node
/usr/share/antimony/nodes/3D/Toruses/torus_y.node
/usr/share/antimony/nodes/3D/Toruses/torus_z.node
/usr/share/antimony/nodes/3D/Volumes
/usr/share/antimony/nodes/3D/Volumes/gyroid.node
/usr/share/antimony/nodes/3D/Volumes/schwarz_d.node
/usr/share/antimony/nodes/3D/Volumes/schwarz_p.node
/usr/share/antimony/nodes/3D/point.node
/usr/share/antimony/nodes/Array
/usr/share/antimony/nodes/Array/array_2d.node
/usr/share/antimony/nodes/Array/array_3d.node
/usr/share/antimony/nodes/Array/array_polar.node
/usr/share/antimony/nodes/CSG
/usr/share/antimony/nodes/CSG/blend.node
/usr/share/antimony/nodes/CSG/clearance.node
/usr/share/antimony/nodes/CSG/copy.node
/usr/share/antimony/nodes/CSG/difference.node
/usr/share/antimony/nodes/CSG/hide.node
/usr/share/antimony/nodes/CSG/intersection.node
/usr/share/antimony/nodes/CSG/invert.node
/usr/share/antimony/nodes/CSG/morph.node
/usr/share/antimony/nodes/CSG/offset.node
/usr/share/antimony/nodes/CSG/shell.node
/usr/share/antimony/nodes/CSG/union.node
/usr/share/antimony/nodes/Color
/usr/share/antimony/nodes/Color/set_color.node
/usr/share/antimony/nodes/Color/set_color_hsb.node
/usr/share/antimony/nodes/Deform
/usr/share/antimony/nodes/Deform/Shear
/usr/share/antimony/nodes/Deform/Shear/shear_cos_x_y.node
/usr/share/antimony/nodes/Deform/Shear/shear_cos_xy_z.node
/usr/share/antimony/nodes/Deform/Shear/shear_x_y.node
/usr/share/antimony/nodes/Deform/Shear/shear_xy_z.node
/usr/share/antimony/nodes/Deform/attract.node
/usr/share/antimony/nodes/Deform/cylinderwrap.node
/usr/share/antimony/nodes/Deform/repel.node
/usr/share/antimony/nodes/Deform/twist.node
/usr/share/antimony/nodes/Export
/usr/share/antimony/nodes/Export/export_heightmap.node
/usr/share/antimony/nodes/Export/export_stl.node
/usr/share/antimony/nodes/Function
/usr/share/antimony/nodes/Function/function_xy.node
/usr/share/antimony/nodes/Function/function_xyz.node
/usr/share/antimony/nodes/Function/map_xyz.node
/usr/share/antimony/nodes/Reflect
/usr/share/antimony/nodes/Reflect/reflect_x.node
/usr/share/antimony/nodes/Reflect/reflect_xy.node
/usr/share/antimony/nodes/Reflect/reflect_xz.node
/usr/share/antimony/nodes/Reflect/reflect_y.node
/usr/share/antimony/nodes/Reflect/reflect_yz.node
/usr/share/antimony/nodes/Reflect/reflect_z.node
/usr/share/antimony/nodes/Rotate
/usr/share/antimony/nodes/Rotate/rotate_x.node
/usr/share/antimony/nodes/Rotate/rotate_y.node
/usr/share/antimony/nodes/Rotate/rotate_z.node
/usr/share/antimony/nodes/Scale
/usr/share/antimony/nodes/Scale/Taper
/usr/share/antimony/nodes/Scale/Taper/taper_x_y.node
/usr/share/antimony/nodes/Scale/Taper/taper_xy_z.node
/usr/share/antimony/nodes/Scale/scale_cos_x_y.node
/usr/share/antimony/nodes/Scale/scale_cos_xy_z.node
/usr/share/antimony/nodes/Scale/scale_x.node
/usr/share/antimony/nodes/Scale/scale_xy.node
/usr/share/antimony/nodes/Scale/scale_xyz.node
/usr/share/antimony/nodes/Scale/scale_y.node
/usr/share/antimony/nodes/Scale/scale_z.node
/usr/share/antimony/nodes/Scale/scale_z_r.node
/usr/share/antimony/nodes/Translate
/usr/share/antimony/nodes/Translate/origin_xy.node
/usr/share/antimony/nodes/Translate/origin_xyz.node
/usr/share/antimony/nodes/Translate/recenter.node
/usr/share/antimony/nodes/Translate/translate.node
/usr/share/appdata/antimony.appdata.xml
/usr/share/applications/antimony.desktop
/usr/share/doc/antimony
/usr/share/doc/antimony/README.md
/usr/share/doc/antimony/SCRIPTING.md
/usr/share/doc/antimony/USAGE.md
/usr/share/icons/antimony
/usr/share/icons/antimony/antimony.svg
/usr/share/licenses/antimony
/usr/share/licenses/antimony/LICENSE
```

**Comment by a2batic** [comment]
Cant open `Add` menu
![screenshot from 2016-07-06 19 49 37](https://cloud.githubusercontent.com/assets/12200504/16621195/07dd2300-43b3-11e6-9b4c-5e964902caff.png)

**Comment by a2batic** [comment]
N
![screenshot from 2016-07-06 19 49 49](https://cloud.githubusercontent.com/assets/12200504/16621372/a39993f0-43b3-11e6-996f-49925a19d27e.png)
o other items available in the 2D, 3D, 2D->3D, 3D->2D and CSG

**Comment by mkeeter** [comment]
Okay, the `Add` menu wasn't even *trying* to populate itself – should be fixed now in the `release/0.9.2b` branch (to be merged into `develop` after the release goes out)

@sagitter 
The root cause of the issue is that Fedora likes to put things in `/usr/{bin,share}`, while other distros put things in `/usr/local/{bin,share}`.  Even Qt's application search paths don't include for `/usr/share`, so I've added a special-cased path of `../share/antimony/nodes` relative to the executable.

Can you pull the latest `release/0.9.2b` commit (`b9f01e14`) and see if that works?
If not, the list of files was very helpful for diagnosing the issue.

**Comment by sagitter** [comment]
>@sagitter
The root cause of the issue is that Fedora likes to put things in /usr/{bin,share}, while other distros put things in /usr/local/{bin,share}. Even Qt's application search paths don't include for /usr/share, so I've added a special-cased path of ../share/antimony/nodes relative to the executable.

>Can you pull the latest release/0.9.2b commit (b9f01e14) and see if that works?
If not, the list of files was very helpful for diagnosing the issue.

`Add` menu is now visible except `2D -> 3D` and `3D -> 2D` sub-menus.

`antimony`'s installed files are:

```
/usr/bin/antimony
/usr/lib/python3.5/site-packages/fab
/usr/lib/python3.5/site-packages/fab/__init__.py
/usr/lib/python3.5/site-packages/fab/__pycache__
/usr/lib/python3.5/site-packages/fab/__pycache__/__init__.cpython-35.opt-1.pyc
/usr/lib/python3.5/site-packages/fab/__pycache__/__init__.cpython-35.pyc
/usr/lib/python3.5/site-packages/fab/__pycache__/shapes.cpython-35.opt-1.pyc
/usr/lib/python3.5/site-packages/fab/__pycache__/shapes.cpython-35.pyc
/usr/lib/python3.5/site-packages/fab/__pycache__/types.cpython-35.opt-1.pyc
/usr/lib/python3.5/site-packages/fab/__pycache__/types.cpython-35.pyc
/usr/lib/python3.5/site-packages/fab/shapes.py
/usr/lib/python3.5/site-packages/fab/types.py
/usr/share/antimony
/usr/share/antimony/nodes
/usr/share/antimony/nodes/2D
/usr/share/antimony/nodes/2D #U2192 3D
/usr/share/antimony/nodes/2D #U2192 3D/Revolve
/usr/share/antimony/nodes/2D #U2192 3D/Revolve/revolve_xy_x.node
/usr/share/antimony/nodes/2D #U2192 3D/Revolve/revolve_xy_y.node
/usr/share/antimony/nodes/2D #U2192 3D/extrude.node
/usr/share/antimony/nodes/2D #U2192 3D/loft.node
/usr/share/antimony/nodes/2D/Circles
/usr/share/antimony/nodes/2D/Circles/circle_center.node
/usr/share/antimony/nodes/2D/Circles/circle_edge.node
/usr/share/antimony/nodes/2D/Rectangles
/usr/share/antimony/nodes/2D/Rectangles/rectangle_center.node
/usr/share/antimony/nodes/2D/Rectangles/rectangle_center_rounded.node
/usr/share/antimony/nodes/2D/Rectangles/rectangle_corner.node
/usr/share/antimony/nodes/2D/Rectangles/rectangle_corner_rounded.node
/usr/share/antimony/nodes/2D/Triangles
/usr/share/antimony/nodes/2D/Triangles/triangle.node
/usr/share/antimony/nodes/2D/Triangles/triangle_right.node
/usr/share/antimony/nodes/2D/point.node
/usr/share/antimony/nodes/2D/polygon.node
/usr/share/antimony/nodes/2D/text.node
/usr/share/antimony/nodes/3D
/usr/share/antimony/nodes/3D #U2192 2D
/usr/share/antimony/nodes/3D #U2192 2D/slice_xy.node
/usr/share/antimony/nodes/3D #U2192 2D/slice_xz.node
/usr/share/antimony/nodes/3D #U2192 2D/slice_yz.node
/usr/share/antimony/nodes/3D/Cones
/usr/share/antimony/nodes/3D/Cones/cone.node
/usr/share/antimony/nodes/3D/Cones/cone_edge.node
/usr/share/antimony/nodes/3D/Cones/cone_height.node
/usr/share/antimony/nodes/3D/Cubes
/usr/share/antimony/nodes/3D/Cubes/cube_center.node
/usr/share/antimony/nodes/3D/Cubes/cube_center_rounded.node
/usr/share/antimony/nodes/3D/Cubes/cube_corner.node
/usr/share/antimony/nodes/3D/Cubes/cube_corner_rounded.node
/usr/share/antimony/nodes/3D/Cylinders
/usr/share/antimony/nodes/3D/Cylinders/cylinder_x.node
/usr/share/antimony/nodes/3D/Cylinders/cylinder_y.node
/usr/share/antimony/nodes/3D/Cylinders/cylinder_z.node
/usr/share/antimony/nodes/3D/Pyramids
/usr/share/antimony/nodes/3D/Pyramids/pyramid_center.node
/usr/share/antimony/nodes/3D/Pyramids/pyramid_edge.node
/usr/share/antimony/nodes/3D/Spheres
/usr/share/antimony/nodes/3D/Spheres/sphere_center.node
/usr/share/antimony/nodes/3D/Spheres/sphere_edge.node
/usr/share/antimony/nodes/3D/Toruses
/usr/share/antimony/nodes/3D/Toruses/torus_x.node
/usr/share/antimony/nodes/3D/Toruses/torus_y.node
/usr/share/antimony/nodes/3D/Toruses/torus_z.node
/usr/share/antimony/nodes/3D/Volumes
/usr/share/antimony/nodes/3D/Volumes/gyroid.node
/usr/share/antimony/nodes/3D/Volumes/schwarz_d.node
/usr/share/antimony/nodes/3D/Volumes/schwarz_p.node
/usr/share/antimony/nodes/3D/point.node
/usr/share/antimony/nodes/Array
/usr/share/antimony/nodes/Array/array_2d.node
/usr/share/antimony/nodes/Array/array_3d.node
/usr/share/antimony/nodes/Array/array_polar.node
/usr/share/antimony/nodes/CSG
/usr/share/antimony/nodes/CSG/blend.node
/usr/share/antimony/nodes/CSG/clearance.node
/usr/share/antimony/nodes/CSG/copy.node
/usr/share/antimony/nodes/CSG/difference.node
/usr/share/antimony/nodes/CSG/hide.node
/usr/share/antimony/nodes/CSG/intersection.node
/usr/share/antimony/nodes/CSG/invert.node
/usr/share/antimony/nodes/CSG/morph.node
/usr/share/antimony/nodes/CSG/offset.node
/usr/share/antimony/nodes/CSG/shell.node
/usr/share/antimony/nodes/CSG/union.node
/usr/share/antimony/nodes/Color
/usr/share/antimony/nodes/Color/set_color.node
/usr/share/antimony/nodes/Color/set_color_hsb.node
/usr/share/antimony/nodes/Deform
/usr/share/antimony/nodes/Deform/Shear
/usr/share/antimony/nodes/Deform/Shear/shear_cos_x_y.node
/usr/share/antimony/nodes/Deform/Shear/shear_cos_xy_z.node
/usr/share/antimony/nodes/Deform/Shear/shear_x_y.node
/usr/share/antimony/nodes/Deform/Shear/shear_xy_z.node
/usr/share/antimony/nodes/Deform/attract.node
/usr/share/antimony/nodes/Deform/cylinderwrap.node
/usr/share/antimony/nodes/Deform/repel.node
/usr/share/antimony/nodes/Deform/twist.node
/usr/share/antimony/nodes/Export
/usr/share/antimony/nodes/Export/export_heightmap.node
/usr/share/antimony/nodes/Export/export_stl.node
/usr/share/antimony/nodes/Function
/usr/share/antimony/nodes/Function/function_xy.node
/usr/share/antimony/nodes/Function/function_xyz.node
/usr/share/antimony/nodes/Function/map_xyz.node
/usr/share/antimony/nodes/Reflect
/usr/share/antimony/nodes/Reflect/reflect_x.node
/usr/share/antimony/nodes/Reflect/reflect_xy.node
/usr/share/antimony/nodes/Reflect/reflect_xz.node
/usr/share/antimony/nodes/Reflect/reflect_y.node
/usr/share/antimony/nodes/Reflect/reflect_yz.node
/usr/share/antimony/nodes/Reflect/reflect_z.node
/usr/share/antimony/nodes/Rotate
/usr/share/antimony/nodes/Rotate/rotate_x.node
/usr/share/antimony/nodes/Rotate/rotate_y.node
/usr/share/antimony/nodes/Rotate/rotate_z.node
/usr/share/antimony/nodes/Scale
/usr/share/antimony/nodes/Scale/Taper
/usr/share/antimony/nodes/Scale/Taper/taper_x_y.node
/usr/share/antimony/nodes/Scale/Taper/taper_xy_z.node
/usr/share/antimony/nodes/Scale/scale_cos_x_y.node
/usr/share/antimony/nodes/Scale/scale_cos_xy_z.node
/usr/share/antimony/nodes/Scale/scale_x.node
/usr/share/antimony/nodes/Scale/scale_xy.node
/usr/share/antimony/nodes/Scale/scale_xyz.node
/usr/share/antimony/nodes/Scale/scale_y.node
/usr/share/antimony/nodes/Scale/scale_z.node
/usr/share/antimony/nodes/Scale/scale_z_r.node
/usr/share/antimony/nodes/Translate
/usr/share/antimony/nodes/Translate/origin_xy.node
/usr/share/antimony/nodes/Translate/origin_xyz.node
/usr/share/antimony/nodes/Translate/recenter.node
/usr/share/antimony/nodes/Translate/translate.node
/usr/share/appdata/antimony.appdata.xml
/usr/share/applications/antimony.desktop
/usr/share/doc/antimony
/usr/share/doc/antimony/README.md
/usr/share/doc/antimony/SCRIPTING.md
/usr/share/doc/antimony/USAGE.md
/usr/share/icons/antimony
/usr/share/icons/antimony/antimony.svg
/usr/share/licenses/antimony
/usr/share/licenses/antimony/LICENSE
```

**Comment by mkeeter** [comment]
Definitely looks like a unicode issue.  Why your previous install put nodes in
```
/usr/share/antimony/nodes/2D → 3D
```
while this install chose
```
/usr/share/antimony/nodes/2D #U2192 3D
```

**Comment by sagitter** [comment]
Can I set `unicode` during build ?

**Comment by mkeeter** [comment]
Yes?  I don't know what that question means.

I'm confused because [this post](https://github.com/mkeeter/antimony/issues/152#issuecomment-230320758) handled unicode paths correctly while [this post](https://github.com/mkeeter/antimony/issues/152#issuecomment-231022205) did not.

**Comment by sagitter** [comment]
>Yes? I don't know what that question means.

I meant if there is a variable that manages `unicode`.

>I'm confused because this post handled unicode paths correctly while this post did not.

They are `0.9.2` build vs `0.9.2b` build.

**Comment by awgrover** [comment]
I'm still having problems:

1. Need python3.3 which Ubuntu 14.04 doesn't have. The error is
```
Could NOT find PythonLibs (missing: PYTHON_LIBRARIES PYTHON_INCLUDE_DIRS)
(Required is at least version "3.3")
```
Install from fkrull/deadsnakes
```
apt-get install -y python-software-properties
add-apt-repository -y ppa:fkrull/deadsnakes
apt-get update
apt-get install -y python3.3 python3.3-dev
```
That seems to work.

2. Compile errors complaining about std=c++11
```
/usr/include/c++/4.9/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support is currently experimental, and must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
```
I did the quick and dirty addition to CMakeLists.txt:
`add_compile_options(-std=c++11)`
Which seems to work (though I get warnings for "c" compiles.

3. Compile errors complaining about std=c11
`../lib/fab/src/formats/stl.c:14:5: error: ‘for’ loop initial declarations are only allowed in C99 or C11 mode`
I did the quick and dirty addition to CMakeLists.txt:
`add_compile_options(-std=c11)`
Which seems to work (though I get warnings for "c++" compiles.

4. And a compile still fail:
```
/usr/bin/c++   -DQT_CONCURRENT_LIB -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_OPENGL_LIB -DQT_WIDGETS_LIB -Wall -Wextra -g -Werror=switch -O3 -DRELEASE -fPIE -Iapp -I../app -isystem /usr/include/python3.3m -isystem /usr/include/x86_64-linux-gnu/python3.3m -I/usr/include/qt5 -I/usr/include/qt5/QtWidgets -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtCore -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++-64 -I/usr/include/qt5/QtOpenGL -I/usr/include/qt5/QtNetwork -I/usr/include/qt5/QtConcurrent -I../lib/graph/inc -I../lib/fab/inc    -D'GITREV="'b9f01e1'+ "' -D'GITTAG=""' -D'GITBRANCH="release/0.9.2b "' -std=c++11 -std=c11 -MMD -MT app/CMakeFiles/antimony.dir/app/app.cpp.o -MF "app/CMakeFiles/antimony.dir/app/app.cpp.o.d" -o app/CMakeFiles/antimony.dir/app/app.cpp.o -c ../app/app/app.cpp
../app/app/app.cpp: In member function ‘QStringList App::nodePaths() const’:
../app/app/app.cpp:67:13: error: ‘AppDataLocation’ is not a member of ‘QStandardPaths’
             QStandardPaths::AppDataLocation))
             ^
```
Here's the output of cmake:
```
-- The C compiler identification is GNU 4.9.3
-- The CXX compiler identification is GNU 4.9.3
-- Check for working C compiler using: Ninja
-- Check for working C compiler using: Ninja -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler using: Ninja
-- Check for working CXX compiler using: Ninja -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Found PythonLibs: /usr/lib/x86_64-linux-gnu/libpython3.3m.so (found suitable version "3.3.6", minimum required is "3.3") 
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.8") 
-- Found PNG: /usr/lib/x86_64-linux-gnu/libpng.so (found version "1.2.50") 
-- Found FLEX: /usr/bin/flex (found version "2.5.35") 
-- Configuring done
-- Generating done
-- Build files have been written to: /home/awgrover/dev/thirdparty/antimony2/build
```
python3.3 -> Python 3.3.6
c++ -> c++ (Ubuntu 4.9.3-8ubuntu2~14.04) 4.9.3

**Comment by sagitter** [comment]
@a2batic @mkeeter 

This problem should be fixed with [release `antimony-0.9.2-0.3`](http://koji.fedoraproject.org/koji/packageinfo?packageID=21348).

Please, leave a feedback on Bodhi: https://bodhi.fedoraproject.org/updates/FEDORA-2016-6cc0e59225

**Comment by mkeeter** [comment]
I'm going to close this issue since @sagitter 's latest release fixes the titular problem.

@awgrover , I've opened up a [new issue](https://github.com/mkeeter/antimony/issues/164) with your problem.
